docs/docs/configuration/cameras.md

```diff
@@ -114,6 +114,7 @@ This list of working and non-working PTZ cameras is based on user feedback.
 | Uniview IPC672LR-AX4DUPK     |      ✅      |      ❌      | Firmware says FOV relative movement is supported, but camera doesn't actually move when sending ONVIF commands                                  |
 | Uniview IPC6612SR-X33-VG     |      ✅      |      ✅      | Leave `calibrate_on_startup` as `False`. A user has reported that zooming with `absolute` is working.                                           |
 | Vikylin PTZ-2804X-I2         |      ❌      |      ❌      | Incomplete ONVIF support                                                                                                                        |
+| Annke CZ504                  |      ✅      |      ✅      | Annke support provide specific firmware ([V5.7.1 build 250227](https://github.com/pierrepinon/annke_cz504/blob/main/digicap_V5-7-1_build_250227.dav)) to fix issue with OnVif "TranslationSpaceFov" |
```

Comment:
This should be closer to the top of the list, as the list is in alphabetical order